dcos/subcommand.py

```diff
@@ -39,15 +37,38 @@ def command_executables(subcommand, dcos_path):
     return executables[0]
 
 
-def list_paths(dcos_path):
-    """List the real path to executable dcos subcommand programs.
+def get_package_commands(package_name):
+    """List the absolute path(s) to executables for a specific dcos subcommand
+
+    :param package_name: package name
+    :type package_name: str
+    :returns: list of all the dcos program paths in package
+    :rtype: [str]
+    """
+    bin_dir = os.path.join(package_dir(package_name),
+                           constants.DCOS_SUBCOMMAND_VIRTUALENV_SUBDIR,
+                           BIN_DIRECTORY)
+
+    executables = []
+    for filename in os.listdir(bin_dir):
+        path = os.path.join(bin_dir, filename)
+
+        if (filename.startswith(constants.DCOS_COMMAND_PREFIX) and
+                _is_executable(path)):
+
+            executables.append(path)
+
+    return executables
+
+
+def list_paths():
+    """List the absolute path to executable dcos subcommand programs.
```

Comment:
I actually don't think "real" meant absolute.  I think it meant "no symbolic links": https://docs.python.org/2/library/os.path.html#os.path.realpath

Maybe change it back